Dockerfile

```diff
@@ -1,17 +1,12 @@
 FROM ubuntu:16.04
-MAINTAINER Event Store LLP <ops@geteventstore.com>
 
-ENV ES_VERSION=5.0.0-1 \
-    DEBIAN_FRONTEND=noninteractive \
```

Comment:
The guidance [here](https://github.com/moby/moby/issues/4032#issuecomment-192327844) (from the issue linked) suggests we should be using `ARG` to set this during the build only?